public/static-checkout/js/scripts.js

```diff
@@ -0,0 +1,313 @@
+/* eslint-disable no-plusplus */
```

Comment:
is eslint in travis or just the precommit hook? if it is the latter, we could remove all the eslint disable lines and commit with the --no-verify option, would save a few lines

Reply:
The "no i++" was disabled in https://github.com/edx/eslint-config/pull/6. The others I am not sure, but with --no-verify we would still fail in travis